ipapython/config.py

```diff
@@ -71,7 +70,17 @@ def check_ip_option(option, opt, value):
     try:
         return CheckedIPAddress(value)
     except Exception as e:
-        raise OptionValueError("option %s: invalid IP address %s: %s" % (opt, value, e))
+        raise OptionValueError("option %s: invalid IP address %s: %s" %
```

Comment:
When you're changing this, why not to change it to the newer `str.format()` way right away?